src/app/newSettings/Sections/Workspace/Teams/TeamsSection.tsx

```diff
@@ -0,0 +1,100 @@
+import { useEffect, useState } from 'react';
+import { t } from 'i18next';
+import { WorkspaceTeamResponse } from '@internxt/sdk/dist/workspaces/types';
+
+import { useAppSelector } from 'app/store/hooks';
+import workspacesSelectors from 'app/store/slices/workspaces/workspaces.selectors';
+import workspacesService from 'app/core/services/workspace.service';
+import errorService from 'app/core/services/error.service';
+import notificationsService, { ToastType } from 'app/notifications/services/notifications.service';
+
+import Section from 'app/newSettings/components/Section';
+import TeamsList from './components/TeamsList';
+import CreateTeamDialog from './components/CreateTeamDialog';
+
+const TeamsSection = ({ onClosePreferences }: { onClosePreferences: () => void }) => {
+  const selectedWorkspace = useAppSelector(workspacesSelectors.getSelectedWorkspace);
+  const isCurrentUserWorkspaceOwner = useAppSelector(workspacesSelectors.isWorkspaceOwner);
+
+  const [teams, setTeams] = useState<WorkspaceTeamResponse>([]);
+  const [createTeamDialogOpen, setCreateTeamDialogOpen] = useState(false);
+  const [newTeamName, setNewTeamName] = useState('');
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    getTeams();
+  }, []);
+
+  const getTeams = async () => {
+    if (selectedWorkspace) {
```

Comment:
I think it would be a better experience if we added a loader while fetching the teams

Reply:
I realized while doing what I am doing now and will be included in the next PR.